en/editor/components/tiledmap.md

```diff
@@ -0,0 +1,85 @@
+# TiledMap Component References
+
+TiledMap is used to display a map in TMX format in the game.
+
+![tiledmap-component](tiledmap/tiledmap-component.png)
+
+Click the **Add Component -> Components -> TileMap** button under the **Inspector** panel to add the TiledMap component to the node.
+
+![](./tiledmap/add_tiledmap.png)
+
+For the script interface of TiledMap, please refer to [TiledMap API](__APIDOC__/zh/classes/TiledMap.html).
+
+## TiledMap Properties
+
+| Property | Description
+| :---------------- | :----------------- |
+| Tmx Asset | Specifies a map asset in `.tmx` format |
+| EnableCulling | Enables culling, which needs to be turned off if the map needs to be rotated or placed in the 3D camera. If the map is not very large, e.g., less than 5000 blocks, then turning off culling reduces the CPU computational load and the GPU renders directly using the cache.
+
+## Detailed Description
+
+- After adding the TiledMap component, drag and drop a map asset in **.tmx** format from the **Assets** panel to the Tmx Asset property to see the map displayed in the scene.
+- When the Tmx Asset property is added to the TiledMap component, nodes corresponding to the Layers in the map are automatically added to the nodes (floor, barrier, and players nodes in the image below). These nodes are added to the TiledLayer component. **Do not delete the TiledLayer component from these Layer nodes**.
+
+    ![](./tiledmap/tiledlayer.png)
+
+- TiledMap component does not support `mapLoaded` callback. It can be used normally in `start` function.
```

Comment:
```suggestion
- TiledMap component does not support the `mapLoaded` callback. It can be used normally in the `start` function.
```

Reply:
But do you mean it can be loaded in the `start` function? I'm not sure what "used normally" means.